src/library/baseexternallibraryfeature.cpp

```diff
@@ -102,10 +102,7 @@ void BaseExternalLibraryFeature::appendTrackIdsFromRightClickIndex(
         return;
     }
 
-    // Qt::UserRole asks TreeItemModel for the TreeItem's data. We need to
-    // use the data because models with nested playlists need to use the
-    // full path/name of the playlist.
-    *pPlaylist = m_lastRightClickedIndex.data(Qt::UserRole).toString();
+    *pPlaylist = m_lastRightClickedIndex.data().toString();
```

Comment:
I have now Idea when this was broken ...

Reply:
Please add at least a debug assertion before dereferencing pPlaylist:
`DEBUG_ASSERT(pPlaylist);`